frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -52,6 +52,24 @@ export class TemplateValidations {
     return recommendedBuses.length === 0 ? allowedBuses : new Set(recommendedBuses);
   };
 
+  checkBusValidationEquality = (otherTempValidations: TemplateValidations): boolean => {
+    // Check if two sets of bus validations are the same - if the allowed and recommended buses are the same
+    const allowedBuses = this.getAllowedBuses();
+    const recommendedBuses = this.getRecommendedBuses();
+    const otherAllowedBuses = otherTempValidations.getAllowedBuses();
+    const otherRecommendedBuses = otherTempValidations.getRecommendedBuses();
+
+    if (
+      allowedBuses.size === otherAllowedBuses.size &&
+      recommendedBuses.size === otherRecommendedBuses.size &&
+      [...allowedBuses].every((bus) => otherAllowedBuses.has(bus)) &&
+      [...recommendedBuses].every((bus) => otherRecommendedBuses.has(bus))
+    ) {
+      return true;
```

Comment:
we can just return the condition boolean

Reply:
done